.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 coverage/
 .DS_Store
+yarn.lock
```

Comment:
can you please remove the lock files from gitignore as it is required for consistent dev setup

Reply:
Thank you for the review. Fixed.
Do I need to update the lock file?

Reply:
No everything is perfect